scripts/verify_env.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+"""
+Used to check missing deployed lambda env variables with local environment variables.
+Does not check
+
+"""
+import os
+import boto3
+import json
+
+
+ssm_client = boto3.client("ssm")
+stages = ['dev', 'integration', 'staging', 'prod']
+
+
+def get_local_lambda_environment_keys():
+    env = ["ADMIN_USER_EMAILS", "DSS_ES_ENDPOINT", "GOOGLE_APPLICATION_CREDENTIALS", "GOOGLE_APPLICATION_SECRETS"]
+    for name in os.environ['EXPORT_ENV_VARS_TO_LAMBDA'].split():
+            env.append(name)
+    return env
+
+
+def get_ssm_lambda_environment(stage=None):
+    if stage is None:
+        stage = os.environ['DSS_DEPLOYMENT_STAGE']
+    parms = ssm_client.get_parameter(
+        Name=f"/{os.environ['DSS_PARAMETER_STORE']}/{stage}/environment"
+    )['Parameter']['Value']
+    return json.loads(parms)
+
+
+def compare_local_stages(stage=None):
+    """Compares local env values to deployed values"""
+    ssm_env = get_ssm_lambda_environment(stage)
+    local_env = get_local_lambda_environment_keys()
+    return compare_env_lists(local_env, ssm_env.keys())
+
+
+def compare_env_lists(l1: list, l2: list):
+    return list(set.symmetric_difference(set(l1), set(l2)))
+
+
+if __name__ == '__main__':
+    print(compare_local_stages())
```

Comment:
Please add text explaining this output. For instance, something like "WARNING: inconsistent local and deployed lambda environment variables:", and list which variables are where.

Reply:
Added text for output + added source for the missing env values in output.
@xbrianh this makes the output not-pretty  although there is a json for the source of the missing env, the warning string makes this non-parsable by `jq`; an alternative would be to add a {"Status": "WARNING" } to the output.